forest/src/cli/wallet_cmd.rs

```diff
@@ -165,10 +165,23 @@ impl WalletCommands {
             }
             Self::List => {
                 let response = wallet_list().await.map_err(handle_rpc_err).unwrap();
+                let default = wallet_default_address()
+                    .await
+                    .map_err(handle_rpc_err)
+                    .unwrap();
 
-                response.iter().for_each(|address| {
-                    println!("{}", address.0);
-                });
+                println!("Address                                    Default  Balance");
+
+                for address in response {
+                    let addr = address.0.to_string();
+                    let def = if addr == default { "X" } else { "" };
+                    let bal = wallet_balance((addr.clone(),))
+                        .await
+                        .map_err(handle_rpc_err)
+                        .unwrap();
```

Comment:
`unwrap` in general is not a nice way to handle errors, would be great if you could change it to e.g. `expect("failed loading the wallet balance")` or something along those lines. At the same time, I can see that there are a bunch of unwraps across the file so it's just a suggestion to make it a bit better.

Reply:
The `unwrap` will never fail (`handle_rpc_err` returns `!`). We really want something like `into_ok` here if it was stable. Or we could refactor the code.

Reply:
I see, it makes sense then. We should probably consider refactoring this in the future, I wouldn't expect `map_` to have such side effects.